.gitlab/ci/on-push.yml

```diff
@@ -7,6 +7,173 @@
   - echo "======= Download configuration ========"
   - ./Tests/scripts/download_demisto_conf.sh  >> $ARTIFACTS_FOLDER/logs/installations.log
 
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+
+run-unittests-and-lint:
```

Comment:
Why are we putting `run-unittests-and-lint` and `run-validations` jobs in `on-push.yml`? I thought the idea was to make them generic jobs that would be inherited from and used in various of our workflows?